WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -321,9 +316,9 @@ - (void)setFollowing:(BOOL)following
 
     ReaderSiteService *siteService = [[ReaderSiteService alloc] initWithManagedObjectContext:self.managedObjectContext];
     if (following) {
-        [siteService followSiteWithID:[siteID integerValue] site:NULL success:successBlock failure:failureBlock];
+        [siteService followSiteWithID:[siteID integerValue] fromTopic:NULL fromPost:NULL success:successBlock failure:failureBlock];
```

Comment:
[Q] Is there a reason why you're using `NULL` instead of `nil`? I thought `NULL` was for C pointers. (https://nshipster.com/nil/)

Reply:
Except that it is not technically incorrect, and the fact that it brings me back in time :) no particular reason. `nil` is specific to Objective-C objects, so I agree it is more suitable here. I'll update the code.